pkgs/by-name/kc/kcl-language-server/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "kcl-language-server";
```

Comment:
There is a name clash with [KCL language](https://www.kcl-lang.io/).

I suggest you name it `kittycad-kcl-lsp`.

(This for top-level name of package.)

Reply:
done

Reply:
Remember to update the directory location/name to `kittycad-kcl-lsp`.